### DISCUSSION TITLE
pip installation and version compatability issue

### DISCUSSION THREAD

**Discussion by C0d3-R3d** [fastapi/fastapi#7263] [Questions] [answered]
I'm getting issue regarding pip. 
in the process of adding or installing module. I got issue with python2 shutdown.
so i upgraded my pip. And now i'm getting version compatibility issue:-

sudo pip3 install pywin32
ERROR: Could not find a version that satisfies the requirement pywin32>=223 (from pypiwin32) (from versions: none)
ERROR: No matching distribution found for pywin32>=223 (from pypiwin32)

sudo pip3 install pywin223
ERROR: Could not find a version that satisfies the requirement pywin223 (from versions: none)
ERROR: No matching distribution found for pywin223

so i tried:- 

sudo pip3 install pypiwin223
ERROR: Could not find a version that satisfies the requirement pypiwin223 (from versions: none)
ERROR: No matching distribution found for pypiwin223

and still getting the issue. 

can anyone help me to fix it? I'm using pycharm too and as usual it still stuck at this position.
I did everything what i got on trough searches or what i can do.. but i still stuck at somewhere. Please help!!
i'm adding screenshot of this issue too.

![Screenshot_2020-08-26_13-11-29](https://user-images.githubusercontent.com/33718703/91335863-00f39080-e79f-11ea-8446-a57a5baea747.png)

Thank you

**Comment by ArcLightSlavik** [comment, marked as answer]
1) This is not related to fastapi
2) Installing this extension in linux does not work (https://github.com/mhammond/pywin32/issues/1372#issuecomment-513989223)
3) If your on windows you need to use .exe binaries and not pip install

**Comment by ycd** [comment]
Is the problem resolved @C0d3-R3d  ? If it is, do you mind closing the issue?

**Comment by tiangolo** [comment]
Thanks for the help here everyone! :clap: :bow: 

If that solves the original problem, then you can close this issue :heavy_check_mark:

**Comment by C0d3-R3d** [comment]
I'm really sorry for not closing the thread and keeping this so long. Totally forget it @tiangolo @ycd  .

Somehow i used windows and it worked there, so i don't know whether it is fixed for linux or not yet, 
Still.. Thanks everyone for help and support. 

Have a nice day ahead.
